pkg/reconciler/taskrun/taskrun.go

```diff
@@ -359,6 +358,20 @@ func (c *Reconciler) prepare(ctx context.Context, tr *v1.TaskRun) (*v1.TaskSpec,
 		}
 	}
 
+	steps, err := resources.GetStepActionsData(ctx, *taskSpec, tr, c.PipelineClientSet)
+	switch {
+	case err != nil:
+		logger.Errorf("Failed to determine StepAction to use for taskrun %s: %v", tr.Name, err)
+		if resources.IsGetTaskErrTransient(err) {
+			return nil, nil, err
+		}
+		tr.Status.MarkResourceFailed(podconvert.ReasonFailedResolution, err)
+		return nil, nil, controller.NewPermanentError(err)
+	default:
+		// Store the fetched StepActions to taskspec
```

Comment:
For consistency in case
```suggestion
		// Store the fetched StepActions to TaskSpec
```